front/ui/ui-core/src/components/inputs/datePicker/__tests__/useCalendar.spec.ts

```diff
@@ -147,19 +147,27 @@ describe('useCalendar', () => {
   });
 
   describe('isToday', () => {
-    it('should return true for today', () => {
-      const today = new Date();
-      expect(defaultResult.current.isToday(today)).toBe(true);
-    });
+    const { result } = renderHook(() =>
+      useCalendar({
+        displayedMonthStartDate,
+        selectedSlot,
+        onDayClick: () => {},
+      })
+    );
```

Comment:
Hum the previous test was not relevant and did not test correctly the function because no `referenceDate` was provided (no `selectableSlot`).
In your commit here, you test again with `useCalendar`, which does not take a `selectableSlot` so you won't fix the test.

I think it would be better to keep using the defaultResult, which is instantiated with a `selectableSlot`